scripts/TVs/add_new_media.py

```diff
@@ -170,10 +171,19 @@ def add_new_media(username=None, tv=None):
         if utils.confirm("Would you like to add another image or video?"):
             media_url = True
             username_invalid = True
+            if username not in students_to_refresh:
+                students_to_refresh.append(username)
         else:
             break
 
         ssh_connection.close()
```

Comment:
This is outside the break, so connection is never closed if you break?

Reply:
solved in [28db17b](https://github.com/timberline-secondary/hackerspace-scripts-2/pull/130/commits/28db17b7a6ec34e3b00ff1e046e6cd1243271c5a)